mmcls/models/necks/gem.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch import Tensor, nn
+from torch.nn import functional as F
+from torch.nn.parameter import Parameter
+
+from ..builder import NECKS
+
+
+def gem(x: Tensor, p: int = 3, eps: float = 1e-6) -> Tensor:
```

Comment:
here `p` is not an `int` parameter value but a `torch.nn.parameter.Parameter` or a `torch.Tensor`